c++/src/kj/string.h

```diff
@@ -278,14 +278,18 @@ class String {
   inline bool operator!=(decltype(nullptr)) const { return content.size() > 1; }
 
   inline bool operator==(const StringPtr& other) const { return StringPtr(*this) == other; }
+#if !__cpp_impl_three_way_comparison
```

Comment:
is the plan to drop these eventually? Maybe we need our own KJ_ alias so that it will be easier to do a cleanup?

Reply:
Yep, I drop them in PR #1732. I considered trying to land that PR first, so I wouldn't add this stuff only to immediately remove it. But, I thought that might take longer to land.